apps/isomorphic-i18n/package.json

```diff
@@ -78,7 +80,7 @@
     "lucide-react": "^0.471.1",
     "mapbox-gl": "^3.5.2",
     "maplibre-gl": "^4.5.0",
-    "next": "14.2.3",
+    "next": "14.1.0",
```

Comment:
Hey Lorenzo, if you update it here, you should update it everywhere. You may have missed it here: `apps/isomorphic/package.json`